### PR TITLE
callback when playlist finishes

### PR DESCRIPTION
**Pull request by MortenHe**
added callback when playlist finishes

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -65,6 +65,11 @@ const createPlayer = () => {
 	const onLine = (line) => {
 		debug('line: ' + line)
 		if (line === 'Starting playback...') return out.emit('track-change')
+
+		//Callback when playlist finishes
+		if (line === "ANS_ERROR=PROPERTY_UNAVAILABLE") {
+			return out.emit('playlist-finished')
```
Comment (derhuerst) [inline index.js]: To keep the event names consistent, please use `playlist-finish`.

File: index.js [hunk 2]
```diff
@@ -79,10 +84,10 @@ const createPlayer = () => {
 	}
 
 	proc.stdout
-	.pipe(byLine.createStream())
-	.on('data', (line) => {
-		onLine(Buffer.isBuffer(line) ? line.toString() : line)
-	})
+		.pipe(byLine.createStream())
```
Comment (derhuerst) [inline index.js]: Please keep the indentation as before.

File: index.js [hunk 3]
```diff
@@ -17,9 +17,9 @@ const createPlayer = () => {
 		'-quiet',
 		'-msglevel', 'all=1:global=4:cplayer=4'
 	], {
-		env: process.env,
-		stdio: ['pipe', 'pipe', 'ignore']
-	})
+			env: process.env,
```
Comment (derhuerst) [inline index.js]: Please keep the indentation as before.

**Review by derhuerst — CHANGES_REQUESTED** [review]
Thanks so far! I only have minor comments.

File: index.js [hunk 4]
```diff
@@ -99,4 +101,4 @@ const createPlayer = () => {
 	return out
 }
 
-module.exports = createPlayer
+module.exports = createPlayer
```
Comment (derhuerst) [inline index.js:104]: almost done! 😉

**Comment by MortenHe** [comment]
i tried again

**Pull request closed without merging**